nautobot/core/templatetags/helpers.py

```diff
@@ -813,6 +813,66 @@ def label_list(value, suffix=""):
     )
 
 
+def get_powerfeed_connection_data(request, instance):
```

Comment:
This doesn't look like a templatetag - I'd just implement it on the view itself if it's not reusable.